method/message/data/createLao.json

```diff
@@ -0,0 +1,110 @@
+{
+    "$comment": "By sending the lao/create message to the organizer’s server’s default channel (“/root”), the main channel of the LAO will be created with the identifier id. At that point, any of the clients, including the organizer, can subscribe to the channel id and send messages over it.",
+    "$comment": "The server is expected to verify the data, such that the last modified timestamp is equal to the creation timestamp, that the timestamp is reasonably recent with respect to the server’s clock, that the attestation is valid, etc.",
+    "$comment": "Upon successful creation, the server/organizer is expected to broadcast the LAO state to all witnesses and clients (see “LAO state broadcast”).",
+    "$schema": "http://json-schema.org/draft/2019-09/schema#",
+    "$id": "http://pop2020.dedis.ch/createLao.json",
+    "type": "object",
+    "properties" : {
+        "jsonrpc": {
+            "const": "2.0"
+        },
+        "method": {
+            "description": "Operation to be performed",
+            "type": "string",
+            "enum": ["publish"]
+        },

```

Comment:
Do we need to repeat this common part every time?

Reply:
I am not sure to understand how this would be possible. 

- If we want the schema to be utilizable by a JsonSchema validator, I don't understand how to do it different (because referencing a higher-level schema (e.g. `generic.json` from the top-level folder) from a lower-level schema (e.g. `createLao.json`) is to my knowledge not possible if we want to then replace the `$ref` in the imported top-level schema). 
- If we want the schema to be human-readable (practical to read), then this would be possible. In this case, should the `message.json` file resemble something like this?

```json
{
	"type": "object",
	"properties": {
		"data": {
			"type": "object",
			"$ref": ???
		},
		"sender": {
			"type": "string", 
			"pattern": "^0x[a-fA-F0-9]+$",
			"$comment": "[HexString] public key of the sender"
		},
		"signature": {
			"type": "string", 
			"pattern": "^0x[a-fA-F0-9]+$",
			"$comment": "[HexString] organizer's signature on data : Sign(Hash(data))"
		},
		// ... message_id and witness_signatures
	}
}
```

I didn't include `title`, `description`, `additionalProperties` and `required` for clarity, but the idea is there. The `???` is a reference to the data object (defined in the `data` folder).